### PR TITLE
Reimplement serialization scope and scope_name

### PR DESCRIPTION
**Pull request by mateomurphy**
I've more or less directly dropped in the code, and I'm open to discussing alternate approaches; I will provide tests once the direction is settled.

### CODE REVIEW

**Comment by JoshSmith** [comment]
This looks pretty straightforward to me.

@kurko what do you think?

**Comment by JoshSmith** [comment]
This is probably the only thing keeping me from upgrading to AMS master.

I'd like thoughts from @kurko or @guilleiguaran or @joaomdmoura before just giving this the :+1:

File: lib/action_controller/serialization.rb [hunk 1]
```diff
@@ -8,6 +8,16 @@ module Serialization
 
     ADAPTER_OPTION_KEYS = [:include, :fields, :root, :adapter]
 
+    included do
+      class_attribute :_serialization_scope
+      self._serialization_scope = :current_user
+    end
+
+    def serialization_scope
+      send(_serialization_scope) if _serialization_scope &&
```
Comment (joaomdmoura) [inline lib/action_controller/serialization.rb]: if possible, use public_send instead :smile:

**Comment by JoshSmith** [comment]
Thanks for help @joaomdmoura :beers:

**Comment by joaomdmoura** [comment]
No worries. You can count on me :beers:

**Pull request merged**